pyproject.toml

```diff
@@ -71,26 +76,29 @@ test = [
 [tool.docstrfmt]
 extend_exclude = ['./docs/examples/']
 
+[tool.hatch.version]
+path = "praw/const.py"
```

Comment:
This issue was fixed in prawcore iirc. I think the version was moved to `__init__.py`.

Reply:
I looked briefly at moving `__version__.py` but it would require `reddit.py` to import from `__init__.py` which would require some import moving around to handle. I didn't like it, so changed the build system instead ;)

Reply:
Works for me!